privaterelay/management/commands/invite_monitor_waitlist.py

```diff
@@ -1,30 +1,88 @@
 from datetime import datetime
+from email.utils import parseaddr
+import logging
 
+from socketlabs.injectionapi.message.basicmessage import BasicMessage
+from socketlabs.injectionapi.message.emailaddress import EmailAddress
+
+from django.conf import settings
 from django.core.management.base import BaseCommand
+from django.template.loader import render_to_string
 
+from emails.utils import get_socketlabs_client, socketlabs_send
 from ...models import Invitations, MonitorSubscriber
 
 
+logger = logging.getLogger('events')
+
+
 class Command(BaseCommand):
     help = 'Connects to Monitor waitlist and creates invitations.'
 
+    def add_arguments(self, parser):
+        parser.add_argument('limit', nargs=1, type=int)
+
     def handle(self, *args, **options):
+        limit = options['limit'][0]
+
         monitor_waitlist = MonitorSubscriber.objects.using('monitor').filter(
             waitlists_joined__email_relay__notified=False
-        )
+        )[:limit]
+
         print("adding %s invitations ..." % monitor_waitlist.count())
+
         for invitee in monitor_waitlist:
             try:
                 Invitations.objects.get(
                     email=invitee.primary_email,
                     active=True,
                 )
-                print("%s already has an active invitation" % invitee.primary_email)
+                print(
+                    "%s already has an active invitation" %
+                    invitee.primary_email
+                )
             except Invitations.DoesNotExist:
                 print("adding %s to invitations" % invitee.primary_email)
-                Invitations.objects.create(
+                invitation = Invitations.objects.create(
                     email=invitee.primary_email,
                     active=True,
                     date_added=datetime.now(),
                     date_redeemed=None
                 )
+
+                # Send email invite
+                sl_message = BasicMessage()
+                sl_message.subject = (
+                    "You're Invited to the Private Relay Beta!"
+                )
+
+                sl_message.html_body = render_to_string(
+                    'emails/beta_invite_html_email.html',
+                    {'email': invitee.primary_email}
+                )
+                sl_message.plain_text_body = render_to_string(
+                    'emails/beta_invite_text_email.txt',
+                    {'email': invitee.primary_email}
+                )
+
+                relay_display_name, relay_from_address = parseaddr(
+                    settings.RELAY_FROM_ADDRESS
+                )
```

Comment:
We can use the `current_site.domain` and create the config value `RELAY_FROM_ADDRESS` in `settings.py` or use this as is but update the .env for each environment. I think either method works and I don't have a strong preference.

Side note: if you decide to configure in `settings.py` instead of `.env`, may I suggest creating a config value to store the `current_domain` to be used in the `context` variable I suggested?

Reply:
We've disabled the admin site in stage & prod environments, so using the Django `Site` objects may be harder than using an environment variable.

Reply:
I'll use an env var that is also put into `settings`.